test/specs/basic.js

```diff
@@ -1,11 +1,31 @@
 const assert = require('assert');
-let should = require('chai').should()
-import loginPage from '../pages/Login.page';
-
-describe('Navigate to archmule page', () => {
-  it('should have the right title', () => {
-    browser.url('https://archmule.com/');
-    const title = browser.getTitle();
-    title.should.be.equal('Archmule');
+let should = require('chai').should();
+
+
+describe('Login and logout of the archmule page', () => {
+  it('should have the right user and password credentials', () => {
+    browser.url('https://archmule.com/login');
+
+    let mailUserTxt = $('#identifier');
+    mailUserTxt.setValue("sirius_779@hotmail.com");
+
+    let passwrdTxt = $('#password');
+    passwrdTxt.setValue('bootcamp');
+
+    let loginButton = $("button[type='submit']");
+    loginButton.click();
+    browser.pause(2000);
+
+    let locator = "img[alt=\"user's avatar\"]";
+    let userAvatar = $(locator);
+    userAvatar.click();
+    browser.pause(2000);
+
+    let logOutBtn = $("button[type='submit']");
+    logOutBtn.click();
+    const url = browser.getUrl();
+    url.should.be.equal('https://archmule.com/');
+
   });
+
```

Comment:
we were supposed to add al least two test cases, the first one when the log in is successful and the other when we cannot log in the app